README.md

```diff
@@ -1,6 +1,14 @@
-# Multiplayer
-Multiplayer mod for RimWorld
+# Rimworld Multiplayer
+Originally created by [Zetrith](https://github.com/Zetrith/Multiplayer), Rimworld Multiplayer is now being updated and maintained by community members.
```

Comment:
I'd suggest we not link to the original GitHub, people will download the wrong releases from the old repository.

Reply:
Fixed